packages/jsActions/web-nanoflow-actions/src/TakePicture.ts

```diff
@@ -637,6 +549,66 @@ export async function TakePicture(picture: mendix.lib.MxObject): Promise<boolean
 
             stream = undefined;
         }
+
+        async function savePicture(videoCanvas: HTMLCanvasElement, onSuccess?: () => void): Promise<void> {
```

Comment:
This doesn't have to be an async function and return `Promise<void>` right? We only start a Promise, don't return it or await it?

Reply:
True, I removed some internal methods that were async.

Reply:
Fixed